index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta name="viewport" content="width=device-width, intial-scale=1" charset="utf-8">
+  <title>Adopt A Dog</title>
+
+  <link rel="stylesheet" type="text/css" href="./styles/index.css">
+
+
+</head>
+
+<body>
+  <header class="col-12">
+    <div class="header-banner">
```

Comment:
Would recommend using flexbox wrap, so on mobile screens the checkout button wraps below the logo